terraform/runner-manager/set_space_egress.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+org="gsa-tts-devtools-prototyping"
+
+usage="
+$0: Set egress rules for given space
+
+Usage:
+  $0 -h
+  $0 -s <SPACE NAME> [-o <ORG NAME>] [-p] [-t]
+
+Options:
+-h: show help and exit
+-s <SPACE NAME>: configure the space to act on. Required
+-o <ORG NAME>: configure the organization to act on. Default: $org
+-p: Add the public egress rules
+-t: Add the trusted egress rules
+"
+
+set -e
+
+space=""
+public=false
+trusted=false
+
+while getopts ":hs:o:pt" opt; do
+  case "$opt" in
+    s)
+      space=${OPTARG}
+      ;;
+    o)
+      org=${OPTARG}
+      ;;
+    p)
+      public=true
+      ;;
+    t)
+      trusted=true
+      ;;
+    h)
+      echo "$usage"
+      exit 0
+      ;;
+  esac
+done
+
+if [[ $space = "" ]]; then
+  echo "$usage"
+  exit 1
+fi
+
+if [[ $public = true ]]; then
+  cf bind-security-group public_networks_egress $org --space $space > /dev/null
```

Comment:
Suggestion (non-blocking): Wrap the names to prevent space/other shenanigans. Also don't silence the output in case this throws an error we should know about.
```suggestion
  cf bind-security-group public_networks_egress "$org" --space "$space"
```

Reply:
Will address as part of #63 (and the output redirection is required for this use-case, though it's possible that `>&2` would work)